sw/device/silicon_creator/lib/boot_svc/BUILD

```diff
@@ -2,39 +2,44 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-load("//rules:cross_platform.bzl", "dual_cc_device_library_of", "dual_cc_library", "dual_inputs")
-
 package(default_visibility = ["//visibility:public"])
 
-dual_cc_library(
+cc_library(
     name = "boot_svc_header",
```

Comment:
There's a little code churn here, maybe squash the third commit into the first?

Reply:
Yeah, the third commit is for @cfrantz -- just to see how it looks without dual_cc_library, will fixup once I hear from him